src/main/java/io/v4guard/plugin/bungee/v4GuardBungee.java

```diff
@@ -15,24 +16,36 @@ public class v4GuardBungee extends Plugin {
 
     @Override
     public void onEnable(){
-        this.getProxy().getConsole().sendMessage("§e[v4guard-plugin] (Bungee) Enabling...");
+        this.getProxy().getConsole().sendMessage(
+                new TextComponent("§e[v4guard-plugin] (Bungee) Enabling...")
```

Comment:
If you don't specifiy the component used, it will send a deprecation warning wich makes poorly source.